### DISCUSSION TITLE
How is the duplicateCount column in the fragment file used in QC metrics?

### DISCUSSION THREAD

**Discussion by daniel-conn17** [stuart-lab/signac#569] [Q&A] [answered]
I'm currently working with a fragment file that doesn't contain the column, duplicateCount 
(the number of PCR duplicate read pairs observed for each fragment). 

I was wondering how duplicateCount is used in the QC metrics: nucleosome banding pattern, 
transcriptional start site (TSS) enrichment score, total number of fragments in peaks,
fraction of fragments in peaks, ratio reads in genomic blacklist regions. 

From my reading of the documentation it seems as though these metrics operate at the level
of fragment reads rather than unique fragments. Thus, the duplicateCount column is used in all
of them. Is that correct? How misleading would it be to use these metrics, assuming that the 
duplicateCount was constant for unique fragments?

**Comment by timoast** [comment, marked as answer]
The PCR duplicate count is not used when computing those metrics (nucleosome signal, TSS enrichment, blacklist region count, etc). Currently all the functions in Signac only use the fragment count information, and the PCR duplicate number is not used. This column is required in the fragment file format, but if it is missing in your case for whatever reason, you could replace it with 1 for all entries for example and it will not affect the results. We could build methods in future that use the PCR duplicate information (assessing the sequencing saturation of the experiment for example).
